Automattic/wp-calypso
# Notifications: Move post title to end of line

#### Changes proposed in this Pull Request

* Combine with changes proposed in D29846-code, this moves the post title to the end of the subject to avoid long post titles pushing the blog name off screen.
* You need a WP.com sandbox to test these changes.

**Before**

{Name} posted {Title} on {Blog}

**After**

{Name} posted on {Blog}
{Title}


#### Testing instructions

* Switch to this PR
* Sandbox `public-api.wordpress.com`
* Apply D29846-code on your sandbox
* Check the Notifications panel on the front end and back end of Calypso. Posts should have the new format.

Fixes #32986
